src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,46 @@
+import { useParams } from 'react-router-dom';
+import React from 'react';
+import { Person } from '../../types/Person';
+import { PersonItem } from '../PersonItem';
+import { SortType } from '../../types/SortType';
+import { SortLink } from '../PeopleFilter/SortLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { selectedPersonSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {Object.values(SortType).map(sorType => (
+            <th key={sorType}>
+              <span className="is-flex is-flex-wrap-nowrap">
+                {sorType.charAt(0).toUpperCase() + sorType.slice(1)}
```

Comment:
Another good way to do such a transformation it's using a CSS rule `text-transform: capitalize;`. But this CSS will make only the first letter capitalized, so it will not work if the text is uppercased, but in your example, it'll be fine.